### DISCUSSION TITLE
Unable to resolve module error

### DISCUSSION THREAD

**Discussion by rgbpokka** [vydimitrov/react-countdown-circle-timer#154] [Q&A] [answered]
I am getting this error despite having installed the package: 

![image](https://user-images.githubusercontent.com/66720042/124456822-bcbd1d80-ddbd-11eb-8275-0fefa7d66c9e.png)

I have no idea what is causing this issue. I'm using react native. Advice is appreciated thank you.

**Comment by vydimitrov** [comment, marked as answer]
Hey, are you sure `react-native-svg` is installed and linked correctly?

**Reply by rgbpokka** [reply]
Oops, I didn't have it installed. It works now thank you!

**Comment by vydimitrov** [comment]
Try to access a component from it outside the timer.
